woocommerce/woocommerce-ios
# [Bundle products in order] Update design of child items in product bundle on order creation screen

Part of: #11250
Part of: #10428


## Description
We want to make some design updates to the product cards on the order creation screen. These are mainly focused on improving the design of child items for product bundles, but some of the changes apply to all product cards, to clean up the overall design:

* For all cards, when the card is expanded:
   * Reduce the product details font size to subheadline with black font in the top section
   * Keep the row with stock details in the top section
   * Replace the price with the SKU in the top section (the price is only shown below, in the expanded details)
   * Remove the top divider (from https://github.com/woocommerce/woocommerce-ios/pull/11268#pullrequestreview-1747159094)
* For child items:
   * Indented 40×40pt thumbnails
   * Reduce the product name font size to subheadline
   * Change the chevron colour to grey

In a separate PR, the child items will be updated to be read-only (to remove options to add discount, remove from order).

## Testing instructions
Prerequisite: the store has [Product Bundles extension](https://woocommerce.com/products/product-bundles/) enabled, and also a bundle product with non-empty items.

- Go to the Orders tab
- Tap `+` to create an order
- Tap `Add Products` and tap a non-bundle product
- Tap a bundle product from the prerequisite
- Configure the bundle
- Add the products to the order
- On the order form, confirm the design is updated with the changes listed above (with nothing looking out of place for bundle or non-bundle products)

## Screenshots
/|Before|After
-|-|-
Collapsed|![Before-Collapsed](https://github.com/woocommerce/woocommerce-ios/assets/8658164/faed3723-ea4d-4c2a-b124-78177597d7fd)|![Design-After-Collapsed](https://github.com/woocommerce/woocommerce-ios/assets/8658164/0a28b846-54a3-4b97-be67-d0868385543c)
Expanded - Non-bundle product|![Before-Product-Expanded](https://github.com/woocommerce/woocommerce-ios/assets/8658164/30772b79-ee0c-47b8-bc89-2827a8014885)|![Design-After-Product-Expanded](https://github.com/woocommerce/woocommerce-ios/assets/8658164/73f7736b-9a18-4130-ba47-325ccbe8cf08)
Expanded - Bundle product|![Before-ProductBundle-Expanded](https://github.com/woocommerce/woocommerce-ios/assets/8658164/cbb13f47-501a-46c6-aee8-582435334ef5)|![Design-After-ProductBundle-Expanded](https://github.com/woocommerce/woocommerce-ios/assets/8658164/e3c28a0f-8a15-4791-b8a8-4bad8ae1bc0e)

---
